sklearn/preprocessing/data.py

```diff
@@ -250,20 +312,74 @@ def fit(self, X, y=None):
         X : array-like, shape [n_samples, n_features]
             The data used to compute the per-feature minimum and maximum
             used for later scaling along the features axis.
+            X cannot be a sparse matrix.
```

Comment:
Not sure if we want to say this here. We don't say it anywhere else like this. The docstring already says X needs to be array-like.